.devcontainer/features/common/install.sh

```
@@ -10,34 +10,60 @@ echo "Current working directory: $(pwd)"
+  "npm/global.json"
 )
 
 GLOBAL_JSON_FILE=""
-for path in ${GLOBAL_JSON_PATHS[@]}; do
-  for file in $path; do
-    if [ -f "$file" ]; then
-      GLOBAL_JSON_FILE="$file"
+for path in "${GLOBAL_JSON_PATHS[@]}"; do
+  # Use find to avoid shell expansion issues with wildcards
+  if [[ "$path" == *"*"* ]]; then
+    found_file=$(find $(dirname "$path") -name "global.json" -type f 2>/dev/null | head -1)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)